docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -1,32 +1,99 @@
-# Forecast quarterly house sales using MindsDB
+# Forecast Quarterly House Sales using MindsDB
 
 ## Introduction
 
-In this short example we will produce forecasts for a multivariate time series.
+In this short example, we will produce forecasts for a multivariate time series.
 
-## Data and setup
+Make sure you have access to a working MindsDB installation - either locally or via [cloud.mindsdb.com](https://cloud.mindsdb.com/).
 
-The dataset we will use is the pre-processed version of the "House Property Sales" [Kaggle competition](https://www.kaggle.com/datasets/htagholdings/property-sales), in particular, the "ma_lga_12345.csv" file, which tracks quarterly moving averages of house sales aggregated by type and amount of bedrooms in each listing.
+## The Data
 
-Make sure you have access to a working MindsDB installation (either local or via cloud.mindsdb.com), and either load it into a table in your database of choice or upload the file directly to the special `FILES` datasource (via SQL or GUI). Once you've done this, proceed to the next step. For the rest of the tutorial we'll assume you've opted for the latter option and uploaded the file with the name `HR_MA`:
+### Connecting the data
 
-```
-SHOW TABLES FROM files;
+There are a couple of ways you can get the data to follow through with this tutorial.
+
+=== "Connecting as a database via `#!sql CREATE DATABASE`"
+
+    You can connect to a demo database that we've prepared for you. It contains the data used throughout this tutorial which is the `#!sql example_db.demo_data.house_sales` table.
+
+    ```sql
+    CREATE DATABASE example_db
+        WITH ENGINE = "postgres",
+        PARAMETERS = {
+            "user": "demo_user",
+            "password": "demo_password",
+            "host": "3.220.66.106",
+            "port": "5432",
+            "database": "demo"
+    }
```

Comment:
Remember to always add ";" On the queries :)

Reply:
I will make these corrections for all queries in the tutorials :)